### DISCUSSION TITLE
Feature Request: Vertical Slider Button

### DISCUSSION THREAD

**Discussion by CptSugarFree** [Clooos/Bubble-Card#756] [Feature requests]
Currently it's not possible to create a vertical slider button without modifying the create.ts files which requires you to install the cards locally and not via HACS. 

I would like a way to create something like the image without the need of modifying the script files. The image was achieved using CSS however the slider still functions the same as the normal slider button and only visually looks like its works vertically. 
![IMG_0414](https://github.com/user-attachments/assets/64d8a487-dd6c-4a40-aa20-53176caaff10)

**Comment by Doni49** [comment]
I don't see a slider in the image you posted.  But I came here specifically to request the ability to create a vertical slider.

**Comment by einfall** [comment]
any news on this?
